File: src/androidTest/java/com/todoroo/astrid/sync/SyncModelTest.java
package com.todoroo.astrid.sync;

import com.todoroo.astrid.data.RemoteModel;
import com.todoroo.astrid.data.TagData;
import com.todoroo.astrid.data.Task;

public class SyncModelTest extends NewSyncTestCase {
	
	public void testCreateTaskMakesUuid() {
		Task task = createTask();
		assertFalse(RemoteModel.NO_UUID.equals(task.getUUID()));
	}

	public void testCreateTagMakesUuid() {
		TagData tag = createTagData();
		assertFalse(RemoteModel.NO_UUID.equals(tag.getUUID()));	
	}

}

File: app/src/androidTest/java/com/todoroo/astrid/sync/SyncModelTest.kt
package com.todoroo.astrid.sync

import org.tasks.data.entity.Task
import dagger.hilt.android.testing.HiltAndroidTest
import kotlinx.coroutines.runBlocking
import org.junit.Assert.assertNotEquals
import org.junit.Test

@HiltAndroidTest
class SyncModelTest : NewSyncTestCase() {

    @Test
    fun testCreateTaskMakesUuid() = runBlocking{
        val task = createTask()
        assertNotEquals(Task.NO_UUID, task.uuid)
    }

    @Test
    fun testCreateTagMakesUuid() = runBlocking{
        val tag = createTagData()
        assertNotEquals(Task.NO_UUID, tag.remoteId)
    }
}